### ISSUE TITLE
NEMA Type controller has issues in switching phases since ver 1.13

### ISSUE THREAD

**Issue by qichaow** [eclipse-sumo/sumo#11931]
The NEMA Type controller in some instances stay on a certain phase combination until the end of the simulation.
@mschrader15 , I think this issue may be introduced during the rewrite of the code. https://github.com/eclipse/sumo/pull/10598. Did you encounter such issues during your simulation runs?

**Comment by mschrader15** [comment]
@qichaow I have not observed that. Can you attach an example?

**Comment by mschrader15** [comment]
@qichaow did you get a chance to test the config file correction I suggested? If so I think we can close this as it was a problem with phase definitions (no capital 'G')

**Comment by qichaow** [comment]
Yes. I tested and was working. Sorry that I forgot to close this.

Qichao

**Comment by namdre** [comment]
Is this something where more warnings from SUMO could be useful (i.e. on phase definitions without capital 'G') ?

**Comment by qichaow** [comment]
Yeah. I think so.

Qichao
